addons/popochiu/editor/config/config.gd

```diff
@@ -1,127 +1,210 @@
-const _DEFAULT_WIPE_OLD_ANIMS_ENABLED = 'popochiu/import/aseprite/wipe_old_animations'
 
+#region Public #####################################################################################
+static func initialize_project_settings():
+	# ---- GUI -------------------------------------------------------------------------------------
+	_initialize_project_cfg(SCALE_GUI, false, TYPE_BOOL)
+	# TODO: Move this to the properties of the 2-click Context-sensitive template or its InventoryBar
+	# 		component
+	_initialize_project_cfg(INVENTORY_ALWAYS_VISIBLE, false, TYPE_BOOL)
+	# TODO: Move this to the properties of the 2-click Context-sensitive template or its SettingsBar
+	# 		component
+	_initialize_project_cfg(TOOLBAR_ALWAYS_VISIBLE, false, TYPE_BOOL)
+	_initialize_project_cfg(FADE_COLOR, Color.BLACK, TYPE_COLOR)
+	_initialize_project_cfg(SKIP_CUTSCENE_TIME, 0.2, TYPE_FLOAT)
+	# ---- Dialogs ---------------------------------------------------------------------------------
+	_initialize_project_cfg(TEXT_SPEED, 0.1, TYPE_FLOAT, PROPERTY_HINT_RANGE, "0.0,0.1")
+	_initialize_project_cfg(AUTO_CONTINUE_TEXT, false, TYPE_BOOL)
+	_initialize_project_cfg(USE_TRANSLATIONS, false, TYPE_BOOL)
+	_initialize_project_cfg(MAX_DIALOG_OPTIONS, 3, TYPE_INT)
+	_initialize_project_cfg(
+		DIALOG_STYLE,
+		DialogStyle.ABOVE_CHARACTER,
+		TYPE_INT,
+		PROPERTY_HINT_ENUM,
+		# TODO: Add the other options: Portrait Above Character, Bubble Above Character
+		"Above Character,Portrait,Caption"
+	)
+	# ---- Inventory -------------------------------------------------------------------------------
+	_initialize_project_cfg(INVENTORY_LIMIT, 0, TYPE_INT)
+	_initialize_project_cfg(INVENTORY_ITEMS_ON_START, [], TYPE_ARRAY,
+		PROPERTY_HINT_TYPE_STRING, "%d/%d:%s" % [TYPE_STRING, PROPERTY_HINT_FILE, "*tscn"]
+	)
+	# ---- Aseprite Importing ----------------------------------------------------------------------
+	_initialize_project_cfg(ASEPRITE_IMPORT_ANIMATION, true, TYPE_BOOL)
+	_initialize_project_cfg(ASEPRITE_LOOP_ANIMATION, true, TYPE_BOOL)
+	_initialize_project_cfg(ASEPRITE_PROPS_VISIBLE, true, TYPE_BOOL)
+	_initialize_project_cfg(ASEPRITE_PROPS_CLICKABLE, true, TYPE_BOOL)
+	_initialize_project_cfg(ASEPRITE_WIPE_OLD_ANIMATIONS, true, TYPE_BOOL)
+	# ---- Pixel game ------------------------------------------------------------------------------
+	_initialize_project_cfg(PIXEL_ART_TEXTURES, false, TYPE_BOOL)
+	_initialize_project_cfg(PIXEL_PERFECT, false, TYPE_BOOL)
 
-var ei := EditorInterface
-var editor_settings: EditorSettings = ei.get_editor_settings()
+	ProjectSettings.save()
 
-var _plugin_icons: Dictionary
 
+# ---- GUI -----------------------------------------------------------------------------------------
+static func is_scale_gui() -> bool:
+	return _get_project_setting(SCALE_GUI, false)
 
-#region Public #####################################################################################
-func initialize_editor_settings():
```

Comment:
Not sure where this one ended up. Are we still going to have editor settings for the plugin?

Reply:
Yes, but now they are in the `res://addons/popochiu/editor/config/editor_config.gd` script.